src/components/redemption/Congratulations.js

```diff
@@ -36,8 +36,8 @@ const Congratulations = ({ chainId, depositAddress }) => {
 }
 
 const mapStateToProps = (state) => ({
-  depositAddress: state.deposit.depositAddress,
-  chainId: state.tbtc.chainId,
+  depositAddress: state.redemption.depositAddress,
+  chainId: state.tbtc.web3.chainId,
```

Comment:
Looks like we probably need to make this change to `deposit/Congratulations.js` as well, perhaps?